.eslintrc.json

```diff
@@ -47,7 +47,7 @@
         "detectObjects": true
       }
     ],
-    "react/jsx-filename-extension": 2,
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
```

Comment:
I'll remove this once the PR from cometkim gets merged.

Reply:
That's been merged now, so you'll get it if you merge/rebase

Reply:
Thanks, done.